ldap/src/integration-test/resources/applicationContext-security.xml

```diff
@@ -0,0 +1,9 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
I can't see where this file is used, could you explain the purpose of it?

Reply:
I have copied the entire file.
it will be deleted.

Reply:
can you advise what to do ?

Reply:
It looks good now, nothing further to do.